tests/e2e/conftest.py

```diff
@@ -159,8 +159,8 @@ def project_dir() -> Iterator[Path]:
 
 
 @pytest.fixture
-@retry(stop=stop_after_attempt(5) | stop_after_delay(5 * 10))
 def cli_runner(project_dir: Path) -> CLIRunner:
+    @retry(stop=stop_after_attempt(5) | stop_after_delay(5 * 10))
```

Comment:
it turned out our retry config didn't work :D

Reply:
lol